cryptol-remote-api/python/cryptol/__init__.py

```diff
@@ -101,7 +101,7 @@ def __init__(self, connection : HasProtocolState, expr : Any) -> None:
         )
 
     def process_result(self, res : Any) -> Any:
-        return res
+        return from_cryptol_arg(res['value'])
```

Comment:
Aside from the usage in `test-cryptol-remote-api.py` which I just updated, all the other uses of `eval` in this repo that I can see don't actually inspect the result, they just compare it it to other results of calling `eval` (e.g. [`test_basics.py`](https://github.com/GaloisInc/cryptol/blob/master/cryptol-remote-api/python/tests/cryptol/test_basics.py#L40)) so their behavior is unaffected.

Reply:
Apologies for the confusion -- I meant it would break code using `eval` that we don't have access to rather than code we can simply patch in our own repo.

Reply:
Ah, I see. Well yes, it definitely could break code. If that's a concern, maybe we just leave this as-is for now – especially since we're going to have a new and better interface available soon anyway.